README.md

```diff
@@ -26,6 +26,12 @@ the following handlers:
   - Type is always set to "suspicious"
   - Metadata field contains to and from addresses
 
+- AE-UMA-UNAUTHORIZED-MINT
+  - Fired when the UMA VotingToken.mint() method is called by any address other than the UMA Voting contract
+  - Severity is always critical
+  - Type is always set to exploit
+  - Metadata field contains VotingToken contract address, caller address, and transaction hash
+
```

Comment:
We may want to also mention that this agent needs traces enabled

Reply:
Good idea, added.